<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simple_tcp_clientClass</class>
 <widget class="QMainWindow" name="simple_tcp_clientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>simple_tcp_client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_addr">
       <property name="text">
        <string>192.168.31.8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_msg">
       <property name="text">
        <string>Hello World!</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_disconnect">
       <property name="text">
        <string>disconnect</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_connect">
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>*TCP/IP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_port">
       <property name="text">
        <string>12800</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>*Port</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btn_send">
       <property name="text">
        <string>send</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Message</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="pte_instruction">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>421</width>
      <height>161</height>
     </rect>
    </property>
    <property name="plainText">
     <string>说明：
1. 点击&lt;connect&gt; 连接到服务器 
2. 点击&lt;disconnect&gt; 断开和服务器的连接
3. 点击&lt;send&gt; 发送信息到服务器
Command窗口显示服务器的指令信息
Received显示服务器发送给客户端的信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Commands:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Recived：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pte_commands">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>421</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pte_recv">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>421</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="simple_tcp_client.qrc"/>
 </resources>
 <connections/>
</ui>
